### PR TITLE
Imageupload

### PR DESCRIPTION
**Pull request by ppiotr191**

### CODE REVIEW

File: src/main/java/com/intive/patronage/toz/model/db/FileUpload.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package com.intive.patronage.toz.model.db;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import java.util.Date;
+import java.util.UUID;
+
+@Entity
+public class FileUpload {
```
Comment (LukaszPiskadlo) [inline src/main/java/com/intive/patronage/toz/model/db/FileUpload.java]: Consider renaming class to File (or Image in our case). Name shouldn't contain action, it causes confusion.

It's possible to create instance of this class without any of its fields initialized (e.g. with no id).

File: src/main/java/com/intive/patronage/toz/service/FileSystemStorageService.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            Files.createDirectories(Paths.get(location + trueLocation));
+        }
+        return trueLocation;
+    }
+    public FileUpload saveFileUpload(String path, String fileExtension){
+        FileUpload fileUpload = new FileUpload();
+        fileUpload.setId(UUID.randomUUID());
+        fileUpload.setDate(new Date());
+        fileUploadRepository.save(fileUpload);
+        path += File.separator + fileUpload.getId() + "." + fileExtension;
```
Comment (macieg-b) [inline src/main/java/com/intive/patronage/toz/service/FileSystemStorageService.java]: To create directory use String.format instead of concatenation

File: src/main/java/com/intive/patronage/toz/service/FileSystemStorageService.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+import java.util.Date;
+import java.util.UUID;
+import java.util.stream.Stream;
+
+@Service
+public class FileSystemStorageService implements StorageService {
+
+    private final Path rootLocation;
+    private String location = "";
+    private final int amountFilesInFolder = 998;
```
Comment (macieg-b) [inline src/main/java/com/intive/patronage/toz/service/FileSystemStorageService.java]: I suggest moving max amount of files in folder to properties or any configuration.

**Comment by ppiotr191** [comment]
I put my changes, but i dont have idea why travis is shouting failure. In my location it's working very well.

**Comment by LukaszPiskadlo** [comment]
@ppiotr191 `StorageProperties` class is not registered as bean and you try to inject it into another class. You can annotate it with `@Component` or use `@EnableConfigurationProperties` in configuration class. Source: https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html#boot-features-external-config-typesafe-configuration-properties

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -91,3 +91,18 @@ databaseChangeLog:
               - column:
                   name: bank_name
                   value: PKO BO II O/Szczecin
+        - createTable:
```
Comment (LukaszPiskadlo) [inline src/main/resources/db/changelog/db.changelog-master.yaml]: This should be in separate changeset
